src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -2846,23 +2859,23 @@ void Fishing_HandleAquariumDialog(Fishing* this, PlayState* play) {
         }
     }
 
-    if (this->unk_1D3 == 0) {
-        if (this->unk_1D4 == 0) {
+    if (this->isAquariumMessage == 0) {
+        if (this->aquariumWaitTimer == 0) {
             this->actor.flags |= ACTOR_FLAG_0;
 
             if (Actor_ProcessTalkRequest(&this->actor, play)) {
-                D_80B7A678 = D_80B7E078;
-                this->unk_1D3 = 1;
+                sFishLengthToWeigh = sFishingRecordLength;
+                this->isAquariumMessage = 1;
             } else {
                 func_8002F2F4(&this->actor, play);
             }
         } else {
-            this->unk_1D4--;
+            this->aquariumWaitTimer--;
             this->actor.flags &= ~ACTOR_FLAG_0;
         }
     } else if (Actor_TextboxIsClosing(&this->actor, play)) {
-        this->unk_1D3 = 0;
-        this->unk_1D4 = 20;
+        this->isAquariumMessage = 0;
```

Comment:
Looks like isAquarumMessage is bool and should be using true/false
(don't know if oot uses `== false` or `! condition` notation)

Reply:
OoT definitely avoids `!=/== false` when it can

Reply:
Bump